irc.html

```diff
@@ -10,24 +10,26 @@ <h1>Channels</h1>
 <div>
   <p>
     Join one of our <abbr title="Internet Relay Chat">IRC</abbr> channels on Freenode:<br/>
+    To chat on #minetest-hub you need to ask an operator for voice. You can't use webchat on #minetest-dev.<br>
```

Comment:
Just one small thing, i think this sounds better, as a native English speaker:
`Webchat cannot be used on #minetest-dev.`

Reply:
All else is fine for me.

Reply:
Fantastic, I agree.